hack/parse-shasums.sh

```diff
@@ -21,7 +21,7 @@ set -e
 
 source hack/common.sh
 
-if [[ ! -f "${DIGESTS_DIR}/bazel-bin/push-virt-operator.digest" ]]; then
+if [[ ! -f "${DIGESTS_DIR}/bazel-bin/push-virt-exportproxy.digest" ]]; then
```

Comment:
without this change, the first `cluster-sync` after updating your repo will will fail because `virt-exportproxy` and `virt-exportserver` have not been previously built but `virt-operator` has.